app/app/tests/utils/utils.py

```diff
@@ -37,14 +37,15 @@ def get_superuser_token_headers(client: TestClient, type_: str = 'access') -> Di
     return headers
 
 
-def to_json(obj: Any) -> Dict:
-    has_relation = len(obj.__mapper__.relationships.items()) > 0
+def to_json(obj: Any, show_relations: Optional[bool] = True) -> Dict:
+    if show_relations and len(obj.__mapper__.relationships.items()) == 0:
+        show_relations = False
 
     result = {}
     for attr, column in obj.__mapper__.c.items():
         result[column.key] = value.isoformat() if isinstance(value := getattr(obj, attr), date) else value
 
-    if has_relation:
+    if show_relations:
```

Comment:
won't it be easier to just check `show_relations and len(obj.__mapper__.relationships.items()) > 0` here instead of above?